asapdiscovery-data/asapdiscovery/data/postera/manifold_data_tags.yaml

```diff
@@ -61,6 +61,12 @@ manifold_outputs:
       units:
       postfix: msk
 
+  - docking-hit:
```

Comment:
@JenkeScheen  we have added tags of the form  `in-silico_SARS-CoV-2-Mpro_docking-hit_msk` as per previous discussions to mark whether something is a hit by `top-n` criteria in large-scale docking.

Reply:
Perfect, I'll make sure to instruct the medchemists to use this bool.